core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -482,11 +484,8 @@ oppia.directive('conversationSkin', [
           };
 
           $scope.submitAnswer = function(answer, interactionRulesService) {
-            // For some reason, answers are getting submitted twice when the
-            // submit button is clicked. This guards against that.
-            if (_answerIsBeingProcessed ||
-                !$scope.isCurrentCardAtEndOfTranscript() ||
-                $scope.activeCard.destStateName) {
+            if (fatigueDetectionService.addSubmission()) {
+              $scope.$broadcast('oppiaFeedbackAvailable');
               return;
             }
```

Comment:
Do you know, I think you might have just solved a years-old problem here :D

That said, might it be worth keeping this check for the sake of belt-and-suspenders, just to avoid e.g. analytics registering multiple answers when it should only register one? We can change the comment accordingly.